gtsfm/bundle/bundle_adjustment.py

```diff
@@ -62,7 +72,13 @@ def run(
             return initial_data
 
         # noise model for measurements -- one pixel in u and v
-        measurement_noise = gtsam.noiseModel.Isotropic.Sigma(IMG_MEASUREMENT_DIM, 1.0)
+        if self._robust_measurement_noise:
+            measurement_noise = gtsam.noiseModel.Robust(
+                gtsam.noiseModel.mEstimator.Huber(1.35),
```

Comment:
can we try with this and the default?

Reply:
The hyperparameter in Huber is defined in terms of number of sigma. Everywhere in GTSAM, the value of 1.345 is used. I found a reference for it: https://myweb.uiowa.edu/pbreheny/uk/764/notes/12-1.pdf. There is no default value. If we want to experiment, I think the hyperparameter to change will be the sigma in the noise model. 1 seems too small

https://github.com/borglab/gtsam/blob/43e8f1e5aeaf11890262722c1e5e04a11dbf9d75/gtsam/linear/NoiseModel.h#L651.